editor/editor_inspector.cpp

```diff
@@ -1353,6 +1353,9 @@ void EditorInspectorSection::_notification(int p_what) {
 	switch (p_what) {
 		case NOTIFICATION_THEME_CHANGED: {
 			update_minimum_size();
+			bg_color = get_theme_color(SNAME("prop_subsection"), EditorStringName(Editor));
+			bg_color.a /= level;
```

Comment:
Setting it here makes the `p_color` argument in `setup()` unnecessary. You should remove that argument.

Reply:
Removal of this from setup will lead to incorrect color until the theme changed:

For example this:
![изображение](https://github.com/godotengine/godot/assets/3036176/101d3446-c9f5-4c25-bdef-28c76bc64e8a)

Instead of this:
![изображение](https://github.com/godotengine/godot/assets/3036176/a3feff0e-29cc-4a7c-89aa-af9c0a6e307a)

Reply:
Seems like the section is added as child before setup, which is what causes this problem.
It would be nice to sort it out eventually, but for a bugfix it's safer to leave it as is.